### ISSUE TITLE
Not working with 0.16.11

### ISSUE THREAD

**Issue by FStefanni** [GerkinDev/typedoc-plugin-tutorials#6]
Hi,

adding additional documentation to TypeDoc is exactly what I need.
Unfortunately this plugin seems to not work with TypeDoc 0.16.11: it seems to crash over a deprecated API.

Can you help with this please?
Here the log:

```
The @Option decorator is deprecated and will be removed in v0.17.
  (Used to register tutorials-map)
The @Option decorator is deprecated and will be removed in v0.17.
  (Used to register tutorials-directory)
Loaded plugin typedoc-plugin-tutorials

Using TypeScript 3.7.5 from node_modules/typedoc/node_modules/typescript/lib
node_modules/typedoc/dist/lib/utils/options/sources/decorator.js:19
                    return this.application.options.getValue(name);
                                                             ^

ReferenceError: name is not defined
    at TutorialsScannerComponent.get [as tutorialsDirectory] (node_modules/typedoc/dist/lib/utils/options/sources/decorator.js:19:62)
    at TutorialsScannerComponent.aggregateConfig (node_modules/typedoc-plugin-tutorials/dist/components/tutorials-scanner-component.js:29:19)
    at TutorialsConverterComponent.begin (node_modules/typedoc-plugin-tutorials/dist/components/tutorials-converter-component.js:19:77)
    at triggerEvents (node_modules/typedoc/dist/lib/utils/events.js:128:43)
    at triggerApi (node_modules/typedoc/dist/lib/utils/events.js:110:13)
    at eventsApi (node_modules/typedoc/dist/lib/utils/events.js:21:18)
    at Converter.trigger (node_modules/typedoc/dist/lib/utils/events.js:264:13)
    at Converter.convert (node_modules/typedoc/dist/lib/converter/converter.js:89:14)
    at CliApplication.convert (node_modules/typedoc/dist/lib/application.js:72:39)
    at CliApplication.bootstrap (node_modules/typedoc/dist/lib/cli.js:45:34)
```

I like also to suggest to propose a pull request to the TypeDoc project (so maybe this code could became part of the core of TypeDoc).
In fact the tutorials support seems a feature requested by many people (there are some issues in TypeDoc about this support).
I believe *if* this will happen, it will be **great**.

Best regards.

**Comment by GerkinDev** [comment]
Hi,

Unfortunately, I've checked out their codebase, and it needs a huge refactor (and they know it). I used this plugin for a couple of my projects, but I have more urgent tasks currently, and I don't wan't to spend hours of refactor & exploration of their new codebase to fix this while I already know they'll change everything for 1.0.0.

So I don't plan doing neither release nor updates until they fix their architecture and move to something more stable & well documented.
